docs/guide/dash-features-proof-of-service.md

```diff
@@ -1,4 +1,10 @@
-# Proof of Service
+```{eval-rst}
+.. meta::
+  :title: Dash Proof of Service
+  :description: The Proof of Service (PoSe) scoring system motivates masternodes to deliver network services, penalizing non-participation by increasing PoSe scores and eventually disqualifying them from payment eligibility. 
+```
+
+Proof of Service
```

Comment:
The heading got moved / lost its heading-ness here somehow

```suggestion
# Proof of Service
```{eval-rst}
.. meta::
  :title: Dash Proof of Service
  :description: The Proof of Service (PoSe) scoring system motivates masternodes to deliver network services, penalizing non-participation by increasing PoSe scores and eventually disqualifying them from payment eligibility. 
```
```